runtime/near-evm-runner/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "near-evm-runner"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+license = "Apache-2.0"
+repository = "https://github.com/nearprotocol/nearcore"
+homepage = "https://github.com/nearprotocol/nearcore"
+
+[dependencies]
+borsh = "0.7.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1"
+hex = "0.4"
+byteorder = "1.0"
+num-bigint = { version = "0.3", default-features = false }
+num-traits = "0.2.12"
+
+sha2 = "0.8"
+sha3 = "0.8"
+rlp = "0.4.2"
+keccak-hash = "0.2.0"
+ripemd160 = "0.9.0"
+libsecp256k1 = "0.3.5"
+
+evm = { git = "https://github.com/paritytech/parity-ethereum", rev = "eed630a002bbebed3a3097127f2483213ff52079" }
+vm = { git = "https://github.com/paritytech/parity-ethereum", rev = "eed630a002bbebed3a3097127f2483213ff52079" }
```

Comment:
Why it's imported twice?

Reply:
These are two different cargos

Reply:
Are you sure? Looks there's only one package there: 
https://github.com/openethereum/openethereum/blob/master/Cargo.toml
And evm = {...} vm = {same} just alias import this package.

Reply:
https://github.com/openethereum/openethereum/tree/master/ethcore/vm
https://github.com/openethereum/openethereum/tree/master/ethcore/evm

Reply:
I see, i should look at https://github.com/openethereum/openethereum/blob/master/ethcore/Cargo.toml instead of https://github.com/openethereum/openethereum/blob/master/Cargo.toml